Bookish/Bookish.Web/Models/BookViewModel.cs

```diff
@@ -14,5 +10,17 @@ public BookViewModel(Book book)
         {
             Book = book;
         }
+
+        public string IsDisabled(BookCopy copy)
```

Comment:
I would expect a method called `IsDisabled` to return a boolean. Something like `GetStatusClass` or `GetDisabledClass` might be more descriptive 🙂